### PR TITLE
feat(onboarding): Switch from manual setup to sentry-wizard for Android and iOS 

### LINKED ISSUE #2877: Update docs & /for pages to use brew + wizard

### LINKED ISSUE #3198: [iOS] Update docs & /for pages to use brew + wizard

### PR DESCRIPTION
**Pull request by romtsn**
* Switches from manual setup to sentry-wizard for Android and iOS in getting started

image

Should be merged after https://github.com/getsentry/sentry-docs/pull/7767

Closes https://github.com/getsentry/sentry-java/issues/2877
Closes https://github.com/getsentry/sentry-cocoa/issues/3198

### CODE REVIEW

File: static/app/gettingStartedDocs/apple/apple-ios.tsx [hunk 1]
```diff
@@ -24,60 +27,114 @@ export const steps = ({
...
+                  "When you're using Fastlane, it will add a Sentry lane for uploading debug symbols"
+                )}
+              </ListItem>
+            </List>
+            <p>
+              {tct(
+                'Alternatively, you can also [manualSetupLink:set up the SDK manually]. You can skip the steps below when using the wizard.',
+                {
+                  manualSetupLink: (
+                    <ExternalLink href="https://docs.sentry.io/platforms/android/manual-setup/" />
```
Comment (brustolin) [inline static/app/gettingStartedDocs/apple/apple-ios.tsx:89]: h: pointing to the Android setup in here

**Review by brustolin — APPROVED** [review]
There is one thing to fix for cocoa. Everything else looks good. You can merge after fixing that.

File: static/app/gettingStartedDocs/apple/apple-ios.tsx (hunk 1 above)
Comment (romtsn) [inline static/app/gettingStartedDocs/apple/apple-ios.tsx:89]:
```suggestion
                    <ExternalLink href="https://docs.sentry.io/platforms/apple/guides/ios/manual-setup/" />
```

File: static/app/gettingStartedDocs/android/android.tsx [hunk 2]
```diff
@@ -24,16 +28,89 @@ export const steps = ({
     description: (
       <p>
         {tct(
-          'Add the [sagpLink:Sentry Android Gradle plugin] to your [app:app] module:',
+          'Add Sentry automatically to your app with the [wizardLink:Sentry wizard].',
```
Comment (romtsn) [inline static/app/gettingStartedDocs/android/android.tsx:31]:
```suggestion
          'Add Sentry automatically to your app with the [wizardLink:Sentry wizard] (call this in your project directory).',
```

File: static/app/gettingStartedDocs/android/android.tsx [hunk 3]
```diff
@@ -24,16 +28,89 @@ export const steps = ({
     description: (
       <p>
         {tct(
-          'Add the [sagpLink:Sentry Android Gradle plugin] to your [app:app] module:',
+          'Add Sentry automatically to your app with the [wizardLink:Sentry wizard] (call this in your project directory).',
```
Comment (romtsn) [inline static/app/gettingStartedDocs/android/android.tsx:31]:
```suggestion
          'Add Sentry automatically to your app with the [wizardLink:Sentry wizard] (call this inside your project directory).',
```

File: static/app/gettingStartedDocs/apple/apple-ios.tsx [hunk 4]
```diff
@@ -24,60 +27,114 @@ export const steps = ({
     description: (
       <p>
         {tct(
-          'We recommend installing the SDK with Swift Package Manager (SPM), but we also support [alternateMethods: alternate installation methods]. To integrate Sentry into your Xcode project using SPM, open your App in Xcode and open [addPackage: File > Add Packages]. Then add the SDK by entering the Git repo url in the top right search field:',
+          'Add Sentry automatically to your app with the [wizardLink:Sentry wizard] (call this in your project directory).',
```
Comment (romtsn) [inline static/app/gettingStartedDocs/apple/apple-ios.tsx:30]:
```suggestion
          'Add Sentry automatically to your app with the [wizardLink:Sentry wizard] (call this inside your project directory).',
```

**Comment by romtsn** [comment]
@smeubank do we wanna get rid of the manual steps entirely similar to what you've done in https://github.com/getsentry/sentry/pull/55844? For now I'm just highlighting that the users are free to skip the steps below when using the wizard

**Review by HazAT — APPROVED** [review]

**Pull request merged**